README.md

```diff
@@ -30,9 +30,12 @@ provide a functioning website on which a phone app may be developed and improved
 ## Technologies
 
 - Java
+- JavaScript
 - Spring Boot
+- Spring Security
 - MySQL
-- Thymeleaf templates
+- Spring Data JPA
+- Thymeleaf templates & extras for Spring Security
```

Comment:
We could add thymeleaf fragments to this if it seems worth noting. I don't want it to sound like we're grasping for any little thing though. What does anyone else think?

Reply:
Also could/should we add Gradle?